src/quote/Quote.ts

```diff
@@ -108,3 +108,81 @@ export function createAgreedQuote(
     claimerSignature: signature,
   }
 }
+
+export function compressCost(quote: IQuote): any {
+  return Object.values(quote.cost)
+}
+
+export function decompressCost(quoteArray: IQuote['cost']): any {
+  const quote = quoteArray
+  return { gross: quote[0], net: quote[1], tax: quote[2] }
+}
+
+export function compressQuote(quote: IQuote): any {
+  return [
+    quote.attesterAddress,
+    quote.cTypeHash,
+    compressCost(quote),
+    quote.currency,
+    quote.termsAndConditions,
+    quote.timeframe,
+  ]
+}
+export function decompressQuote(quoteArray: any): IQuote {
+  const quote = quoteArray
+  return {
+    attesterAddress: quote[0],
+    cTypeHash: quote[1],
+    cost: decompressCost(quote[2]),
+    currency: quote[3],
+    termsAndConditions: quote[4],
+    timeframe: quote[5],
+  }
+}
+
+export function compressAttesterSignedQuote(
+  attesterSignedQuote: IQuoteAttesterSigned
+): any {
```

Comment:
Change request:
You're returning an array, not `any`

Reply:
Updated